client/src/document/writer-toolbar/edit-buttons.scss

```diff
@@ -1,4 +1,4 @@
-.edit-this-page {
```

Comment:
On second thought; perfection is the enemy of the good. 
But now's the chance to spend a couple of minutes up-front and pick a good name because it's hard to change once it's too late.

Reply:
Definitely open for suggestions, but not yet convinced we found something better than `EditButtons` yet. I guess button is too narrow, but the edit part seems correct. Maybe `EditActions`?